crowbar_framework/app/views/installer/upgrades/restore.haml

```diff
@@ -36,6 +43,4 @@
 
             .col-lg-9.text-right
               .btn-group{ role: "group" }
-                = restore_button
-              .btn-group{ role: "group" }
                 = upgrade_continue_button(true)
```

Comment:
Hm, now the "Restore" button is removed and the continue button is disabled. I can't get any further here.